dockerfiles/enclave_local_build.Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM ghcr.io/edgelesssys/ego-dev:latest
 
 # build the enclave from the current branch
 RUN mkdir /home/obscuro-playground
-COPY ./ /home/obscuro-playground
+COPY . /home/obscuro-playground
```

Comment:
This duplicates https://github.com/obscuronet/go-obscuro/pull/223, but I needed it for this PR.